### PR TITLE
Change 10260 temporal tables removal

### LINKED ISSUE #10260: Replace temporal tables postgres extension with PL/pgSQL function [5]
### Problem Description
The temporal tables extension is no longer maintained and only support Postgres up to version 10 (which is only supported until the end of this year): https://github.com/nearform/temporal_tables
In addition it complicates the server setup process and makes it impossible to run SORMAS as a hosted service when the provided Postgres instance does not support custom extensions.

### Proposed Change
https://github.com/nearform/temporal_tables provides a PL/pgSQL function that replicates the same functionality. Based on their performance tests this is 16x slower than the C implementation, which is most likely related to a missing cache functionality.
From my understanding we don't have a performance problem related to writing history data when inserting new data into the database. The performance problems we are currently facing when persisting data are based on other factors - see #4417.

- [x] Include the provided function script into SORMAS backend
- [x] Add statements to create the function both for newly setup systems and for running systems
- [x] Remove the temporal tables **extension** from all setup instructions
- [x] Remove check from StartupShutdownService
- [x] Provide a draft pull request for SORMAS-Docker to adjust it accordingly --> https://github.com/hzi-braunschweig/SORMAS-Docker/pull/485
- [ ] Release notes need to contain information on SQL that needs to be executed.

### Acceptance Criteria
* Existing and newly setup servers no longer rely on the temporal tables extension
* History tables still work as before

### Additional Information

Command to change owner of existing `versioning` trigger function. **Needs to be called before deploying SORMAS into the server.**

```
-- versioning function will be replaced during server backend startup
ALTER FUNCTION public.versioning() OWNER TO sormas_user;
```

Command to drop extension on existing systems. Can be run **after** server was deployed for the first time: 

```
-- needed because `ALTER FUNCTION ... NO DEPENDS ON EXTENSION` is not possible in Postgres 10 and below
DELETE FROM pg_depend
WHERE deptype = 'e'
  AND refobjid = (SELECT oid FROM pg_extension WHERE extname = 'temporal_tables')
  and objid = (SELECT oid FROM pg_proc WHERE proname = 'versioning');

DROP EXTENSION IF EXISTS temporal_tables;
```

### PR DESCRIPTION
**Pull request by MartinWahnschaffe**
Fixes #10260

I'd suggest to merge this once the 1.79 release is done.

I also need to prepare a pull request for the docker related changes or discuss with Frank whether they will take care of the needed changes.

### CODE REVIEW

**Comment by MartinWahnschaffe** [comment]
@JonasCir I checked and the script is not released to npm or any other package management system (https://github.com/nearform/temporal_tables/blob/master/.github/workflows/release.yml is not configured accordingly). 
We could download the released source from github, but would then have to extract the needed sql files from the zip - all via maven. This would be quite tricky and it would still not give us the benefit of automatically seeing that there is a dependency with a new version that could be updated.

So I'd go this way: 

- [ ] Add the license as a header to the taken script files
- [ ] Add a comment to the maven pom, next to the postgres dependency that states which version is used, so we have at least a hint in the maven file

**Comment by JonasCir** [comment]
Ok let's do that :)

**Review by JonasCir — CHANGES_REQUESTED** [review]
- License in `docs/3rd_PARTY_ACK.md`
- owner of `versioning` function has to be set to sormas user
- also for SORMAS-Docker

**Comment by MartinWahnschaffe** [comment]
https://github.com/hzi-braunschweig/SORMAS-Project/issues/10260#issuecomment-1428067327

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11380

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11380

**Comment by MartinWahnschaffe** [comment]
Related stack trace of docker in S2S-Testbed:

> sormas_a_1    | [#|2023-02-13T17:24:56.404+0100|WARNING|Payara 5.2021.1|javax.enterprise.ejb.container|_ThreadID=1;_ThreadName=main;_TimeMillis=1676305496404;_LevelValue=900;_MessageID=AS-EJB-00056;|
> sormas_a_1    |   A system exception occurred during an invocation on EJB UpdateQueryTransactionWrapper, method: public int de.symeda.sormas.backend.common.StartupShutdownService$UpdateQueryTransactionWrapper.executeUpdate(de.symeda.sormas.backend.common.StartupShutdownService$UpdateQueryTransactionWrapper$TargetDb,java.lang.String)|#]
> sormas_a_1    |
> sormas_a_1    | [#|2023-02-13T17:24:56.404+0100|WARNING|Payara 5.2021.1|javax.enterprise.ejb.container|_ThreadID=1;_ThreadName=main;_TimeMillis=1676305496404;_LevelValue=900;|
> sormas_a_1    |   javax.ejb.EJBException: org.hibernate.exception.SQLGrammarException: could not execute statement
> sormas_a_1    |         at com.sun.ejb.containers.EJBContainerTransactionManager.processSystemException(EJBContainerTransactionManager.java:723)
> sormas_a_1    |         at com.sun.ejb.containers.EJBContainerTransactionManager.completeNewTx(EJBContainerTransactionManager.java:652)
> sormas_a_1    |         at com.sun.ejb.containers.EJBContainerTransactionManager.postInvokeTx(EJBContainerTransactionManager.java:492)
> sormas_a_1    |         at com.sun.ejb.containers.BaseContainer.postInvokeTx(BaseContainer.java:4592)
> sormas_a_1    |         at com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2125)
> sormas_a_1    |         at com.sun.ejb.containers.BaseContainer.postInvoke(BaseContainer.java:2095)
> sormas_a_1    |         at com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:220)
> sormas_a_1    |         at com.sun.ejb.containers.EJBLocalObjectInvocationHandlerDelegate.invoke(EJBLocalObjectInvocationHandlerDelegate.java:90)
> sormas_a_1    |         at com.sun.proxy.$Proxy603.executeUpdate(Unknown Source)
> sormas_a_1    |         at de.symeda.sormas.backend.common.__EJB31_Generated__StartupShutdownService$UpdateQueryTransactionWrapper__Intf____Bean__.executeUpdate(Unknown Source)
> sormas_a_1    |         at de.symeda.sormas.backend.common.StartupShutdownService.createVersioningFunction(StartupShutdownService.java:757)
> sormas_a_1    |         at de.symeda.sormas.backend.common.StartupShutdownService.startup(StartupShutdownService.java:217)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.BeanCallbackInterceptor.intercept(InterceptorManager.java:1022)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.CallbackChainImpl.invokeNext(CallbackChainImpl.java:72)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.CallbackInvocationContext.proceed(CallbackInvocationContext.java:204)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.init(SystemInterceptorProxy.java:125)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.CallbackInterceptor.intercept(InterceptorManager.java:978)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.CallbackChainImpl.invokeNext(CallbackChainImpl.java:72)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.CallbackInvocationContext.proceed(CallbackInvocationContext.java:204)
> sormas_a_1    |         at org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:81)
> sormas_a_1    |         at org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.CallbackInterceptor.intercept(InterceptorManager.java:978)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.CallbackChainImpl.invokeNext(CallbackChainImpl.java:72)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:418)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:381)
> sormas_a_1    |         at com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:2062)
> sormas_a_1    |         at com.sun.ejb.containers.AbstractSingletonContainer.createSingletonEJB(AbstractSingletonContainer.java:529)
> sormas_a_1    |         at com.sun.ejb.containers.AbstractSingletonContainer.access$000(AbstractSingletonContainer.java:81)
> sormas_a_1    |         at com.sun.ejb.containers.AbstractSingletonContainer$SingletonContextFactory.create(AbstractSingletonContainer.java:687)
> sormas_a_1    |         at com.sun.ejb.containers.AbstractSingletonContainer.instantiateSingletonInstance(AbstractSingletonContainer.java:421)
> sormas_a_1    |         at org.glassfish.ejb.startup.SingletonLifeCycleManager.initializeSingleton(SingletonLifeCycleManager.java:219)
> sormas_a_1    |         at org.glassfish.ejb.startup.SingletonLifeCycleManager.initializeSingleton(SingletonLifeCycleManager.java:180)
> sormas_a_1    |         at org.glassfish.ejb.startup.SingletonLifeCycleManager.doStartup(SingletonLifeCycleManager.java:159)
> sormas_a_1    |         at org.glassfish.ejb.startup.EjbApplication.start(EjbApplication.java:171)
> sormas_a_1    |         at org.glassfish.internal.data.EngineRef.start(EngineRef.java:123)
> sormas_a_1    |         at org.glassfish.internal.data.ModuleInfo.start(ModuleInfo.java:293)
> sormas_a_1    |         at org.glassfish.internal.data.ApplicationInfo.start(ApplicationInfo.java:364)
> sormas_a_1    |         at com.sun.enterprise.v3.server.ApplicationLifecycle.initialize(ApplicationLifecycle.java:623)
> sormas_a_1    |         at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:580)
> sormas_a_1    |         at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:556)
> sormas_a_1    |         at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:552)
> sormas_a_1    |         at java.base/java.security.AccessController.doPrivileged(Native Method)
> sormas_a_1    |         at java.base/javax.security.auth.Subject.doAs(Subject.java:361)
> sormas_a_1    |         at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:551)
> sormas_a_1    |         at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:582)
> sormas_a_1    |         at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:574)
> sormas_a_1    |         at java.base/java.security.AccessController.doPrivileged(Native Method)
> sormas_a_1    |         at java.base/javax.security.auth.Subject.doAs(Subject.java:361)
> sormas_a_1    |         at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:573)
> sormas_a_1    |         at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1497)
> sormas_a_1    |         at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:120)
> sormas_a_1    |         at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1879)
> sormas_a_1    |         at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1755)
> sormas_a_1    |         at com.sun.enterprise.admin.cli.embeddable.CommandExecutorImpl.executeCommand(CommandExecutorImpl.java:183)
> sormas_a_1    |         at com.sun.enterprise.admin.cli.embeddable.CommandExecutorImpl.run(CommandExecutorImpl.java:96)
> sormas_a_1    |         at fish.payara.boot.runtime.BootCommand.execute(BootCommand.java:69)
> sormas_a_1    |         at fish.payara.boot.runtime.BootCommands.executeCommands(BootCommands.java:136)
> sormas_a_1    |         at fish.payara.boot.runtime.BootCommands.executeCommands(BootCommands.java:130)
> sormas_a_1    |         at com.sun.enterprise.glassfish.bootstrap.GlassFishMain$Launcher.doBootCommands(GlassFishMain.java:294)
> sormas_a_1    |         at com.sun.enterprise.glassfish.bootstrap.GlassFishMain$Launcher.launch(GlassFishMain.java:133)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.enterprise.glassfish.bootstrap.GlassFishMain.main(GlassFishMain.java:109)
> sormas_a_1    |         at com.sun.enterprise.glassfish.bootstrap.ASMain.main(ASMain.java:54)
> sormas_a_1    | Caused by: javax.persistence.PersistenceException: org.hibernate.exception.SQLGrammarException: could not execute statement
> sormas_a_1    |         at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:154)
> sormas_a_1    |         at org.hibernate.internal.ExceptionConverterImpl.convert(ExceptionConverterImpl.java:181)
> sormas_a_1    |         at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1705)
> sormas_a_1    |         at de.symeda.sormas.backend.common.StartupShutdownService$UpdateQueryTransactionWrapper.executeUpdate(StartupShutdownService.java:1044)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at org.glassfish.ejb.security.application.EJBSecurityManager.runMethod(EJBSecurityManager.java:588)
> sormas_a_1    |         at org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:408)
> sormas_a_1    |         at com.sun.ejb.containers.BaseContainer.invokeBeanMethod(BaseContainer.java:4826)
> sormas_a_1    |         at com.sun.ejb.EjbInvocation.invokeBeanMethod(EjbInvocation.java:665)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:834)
> sormas_a_1    |         at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)
> sormas_a_1    |         at de.symeda.sormas.backend.util.ValidationConstraintViolationInterceptor.handleValidationConstraintViolation(ValidationConstraintViolationInterceptor.java:35)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)
> sormas_a_1    |         at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)
> sormas_a_1    |         at de.symeda.sormas.backend.util.RightsAllowedInterceptor.checkRightsAllowed(RightsAllowedInterceptor.java:111)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)
> sormas_a_1    |         at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)
> sormas_a_1    |         at de.symeda.sormas.backend.util.PerformanceLoggingInterceptor.logInvokeDuration(PerformanceLoggingInterceptor.java:47)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)
> sormas_a_1    |         at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)
> sormas_a_1    |         at de.symeda.sormas.backend.audit.AuditLoggerInterceptor.logAudit(AuditLoggerInterceptor.java:158)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)
> sormas_a_1    |         at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.doCall(SystemInterceptorProxy.java:163)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.SystemInterceptorProxy.aroundInvoke(SystemInterceptorProxy.java:140)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)
> sormas_a_1    |         at com.sun.ejb.EjbInvocation.proceed(EjbInvocation.java:615)
> sormas_a_1    |         at org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:72)
> sormas_a_1    |         at org.jboss.weld.module.ejb.SessionBeanInterceptor.aroundInvoke(SessionBeanInterceptor.java:52)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeInterceptor.intercept(InterceptorManager.java:888)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.AroundInvokeChainImpl.invokeNext(InterceptorManager.java:833)
> sormas_a_1    |         at com.sun.ejb.containers.interceptors.InterceptorManager.intercept(InterceptorManager.java:375)
> sormas_a_1    |         at com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4798)
> sormas_a_1    |         at com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4786)
> sormas_a_1    |         at com.sun.ejb.containers.EJBLocalObjectInvocationHandler.invoke(EJBLocalObjectInvocationHandler.java:212)
> sormas_a_1    |         ... 72 more
> sormas_a_1    | Caused by: org.hibernate.exception.SQLGrammarException: could not execute statement
> sormas_a_1    |         at org.hibernate.exception.internal.SQLStateConversionDelegate.convert(SQLStateConversionDelegate.java:103)
> sormas_a_1    |         at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:37)
> sormas_a_1    |         at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113)
> sormas_a_1    |         at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99)
> sormas_a_1    |         at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:200)
> sormas_a_1    |         at org.hibernate.engine.query.spi.NativeSQLQueryPlan.performExecuteUpdate(NativeSQLQueryPlan.java:107)
> sormas_a_1    |         at org.hibernate.internal.SessionImpl.executeNativeUpdate(SessionImpl.java:1549)
> sormas_a_1    |         at org.hibernate.query.internal.NativeQueryImpl.doExecuteUpdate(NativeQueryImpl.java:299)
> sormas_a_1    |         at org.hibernate.query.internal.AbstractProducedQuery.executeUpdate(AbstractProducedQuery.java:1696)
> sormas_a_1    |         ... 136 more
> sormas_a_1    | Caused by: org.postgresql.util.PSQLException: ERROR: must be owner of function versioning
> sormas_a_1    |         at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2676)
> sormas_a_1    |         at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2366)
> sormas_a_1    |         at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:356)
> sormas_a_1    |         at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:490)
> sormas_a_1    |         at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:408)
> sormas_a_1    |         at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:181)
> sormas_a_1    |         at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:149)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at org.postgresql.ds.PGPooledConnection$StatementHandler.invoke(PGPooledConnection.java:441)
> sormas_a_1    |         at com.sun.proxy.$Proxy762.executeUpdate(Unknown Source)
> sormas_a_1    |         at com.sun.gjc.spi.base.PreparedStatementWrapper.executeUpdate(PreparedStatementWrapper.java:127)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> sormas_a_1    |         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> sormas_a_1    |         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> sormas_a_1    |         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
> sormas_a_1    |         at com.sun.gjc.spi.jdbc40.ProfiledConnectionWrapper40$1.invoke(ProfiledConnectionWrapper40.java:457)
> sormas_a_1    |         at com.sun.proxy.$Proxy763.executeUpdate(Unknown Source)
> sormas_a_1    |         at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:197)
> sormas_a_1    |         ... 140 more
> sormas_a_1    | |#]
>

File: docs/SERVER_UPDATE.md [hunk 1]
```diff
@@ -18,6 +13,46 @@ Note: You can skip this step if you've just set up your SORMAS server and have a
...
+```SQL
+-- needed because `ALTER FUNCTION ... NO DEPENDS ON EXTENSION` is not possible in Postgres 10 and below
+DELETE FROM pg_depend
+WHERE deptype = 'e'
+  AND refobjid = (SELECT oid FROM pg_extension WHERE extname = 'temporal_tables')
+  and objid = (SELECT oid FROM pg_proc WHERE proname = 'versioning');
+
+DROP EXTENSION IF EXISTS temporal_tables;
+```
+
```
Comment (JonasCir) [inline docs/SERVER_UPDATE.md:55]: Can we take away the ownership of the `sormas_user` now or do we need to keep it after the extension was dropped?

**Review by JonasCir — CHANGES_REQUESTED** [review]

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11380

File: docs/SERVER_UPDATE.md (hunk 1 above)
Comment (MartinWahnschaffe) [reply docs/SERVER_UPDATE.md:55]: No, the user should keep the ownership. It actually makes a lot of sense to have the whole functionality in the ownership of this user. All the triggers and some other functions related to this are also in it's ownership.

**Pull request merged**